bin/node/runtime/src/lib.rs

```diff
@@ -132,7 +132,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 2,
-	state_version: 1,
+	state_version: 0,
```

Comment:
Why `0`?

Reply:
Update from 0 to 1 put the state in a hybride mode until we execute a migration. For more details check `pallet_state_trie_migration` in our [notion](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#8484d897223540f6af64fe918df33d4c)

Reply:
I checked. For now we do not apply it and this is why we can stay with `state_version` 0. Correct?

Reply:
Correct